front/src/applications/stdcm/styles/_railwayManagerModal.scss

```diff
@@ -0,0 +1,272 @@
+.send-to-railway-manager {
+  h2 {
+    font-size: 24px;
+    font-weight: 600;
+    padding: 24px 32px 20px;
+    box-shadow: 0 1px 0 0 rgba(0, 0, 0, 0.25);
+    background-color: var(--white);
+    margin-bottom: 1px;
+  }
+
+  // override style from ui-core input
+  .custom-field {
+    label {
+      margin-bottom: 0;
+    }
+  }
+
+  .modal-contents {
+    background-color: var(--ambientA10);
+    padding: 22px 32px 80px;
+
+    > p {
+      max-width: 629px;
+      color: var(--grey80);
+      margin-bottom: 1rem;
+    }
+
+    section {
+      margin-bottom: 2rem;
+
+      h3 {
+        font-family: 'IBM Plex Sans';
+        font-size: 18px;
+        font-weight: 700;
+        margin-bottom: 1rem;
```

Comment:
margin should be in px. You can refer to our [guidelines](https://osrd.fr/fr/docs/guides/contribute/contribute-code/frontend-conventions/) here